src/dataflow/src/logging/materialized.rs

```diff
@@ -38,6 +39,34 @@ pub enum MaterializedEvent {
         /// Globally unique identifier for the source on which the dataflow depends.
         source: GlobalId,
     },
+    /// Tracks statistics for a particular Kafka consumer / partition pair
+    /// Reference: https://github.com/edenhill/librdkafka/blob/master/STATISTICS.md
+    KafkaConsumerInfo {
+        /// Kafka name for the consumer
+        consumer_name: String,
+        /// Materialize source identifier
+        source_id: SourceInstanceId,
```

Comment:
I think that this will generate a string that has a global_id and instance_id inline as e.g. `u0/1` where the global id tracks the source, and the instance ID tracks the per-worker/per-instantiation key.

I think it'd be better to split these out into two columns, because then the `source_id` here would be directly comparable to the `GlobalID`s in other `MaterializedEvent` variants.

Reply:
I noticed that but didn't see any easy way to split this out. Let me look again!

Reply:
Both the fields are pub, I think just renaming this to `source: GlobalId, source_instance: u64` should do it.

Reply:
Done! I missed that SourceInfo does `&id.source_id.to_string()`. Updated kafka_consumer_info to do the same.